backend/src/openarchiefbeheer/api/zaken/views.py

```diff
@@ -0,0 +1,49 @@
+from django.utils.translation import gettext_lazy as _
+
+from drf_spectacular.utils import extend_schema
+from requests.exceptions import RequestException
+from rest_framework.renderers import JSONRenderer
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from zgw_consumers.client import build_client
+from zgw_consumers.constants import APITypes
+from zgw_consumers.models import Service
+
+from openarchiefbeheer.destruction.utils import process_zaken
+
+
+@extend_schema(
+    summary=_("List zaken"),
+    description=_(
+        "Retrieve zaken using the configured ZRC service. "
+        "Zaken already included in a destruction list are not returned. "
+        "For information over the query parameters accepted and the schema of the response, look at the "
+        "'/zaken/api/v1/zaken' list endpoint of Open Zaak."
+    ),
+)
+class ZakenView(APIView):
+    # We don't want the CamelCase renderer to alter the responses from Open Zaak
+    renderer_classes = (JSONRenderer,)
+
+    def get(self, request: Request) -> Response:
+        zrc_service = Service.objects.get(api_type=APITypes.zrc)
+        zrc_client = build_client(zrc_service)
+
+        with zrc_client:
+            response = zrc_client.get(
+                "zaken",
+                headers={"Accept-Crs": "EPSG:4326"},
+                params=request.query_params,
+            )
+
+        try:
+            response.raise_for_status()
+        except RequestException:
+            return Response({"error": response.json()}, status=response.status_code)
+
+        response_data = response.json()
+
+        response_data["results"] = process_zaken(response_data["results"])
```

Comment:
If i understand this correctly (I might be wrong):

- The request is relayed to OZ
- OZ responds with a paginated response including meta information like `count`
- We use filtering on our end to exclude items already in a list

Questions: 
 - Does this mean the `count` field (and possibly others) may be off? Or is the pagination created/updated on our end?
 -  Is it possible that the filtering may result in incorrect page (sizes).

Reply:
I think that updating a set invalidates the pagination and solving this correctly would require to repaginate the entire set which is problematic because (for good reasons) we receive only a fragment from a paginated response from OZ.

I can't think of a simple solution to fix this, lets discuss this together...

Reply:
Discussed: For now remove filtering. In a new issue, tackle this problem with caching the cases from OZ.

Reply:
https://github.com/maykinmedia/open-archiefbeheer/issues/6